content/en/continuous_integration/tests/_index.md

```diff
@@ -33,6 +33,25 @@ cascade:
 
 [Test Visibility][1] provides a test-first view into your CI health by displaying important metrics and results from your tests. It can help you investigate performance problems and test failures that concern you the most because you work on the related code, not because you maintain the pipelines they are run in.
 
+### Supported features
+|                                         | .NET | Java |   Javascript   | Python | Ruby | Swift | JUnit Xml |
+|-----------------------------------------|:----:|:----:|:--------------:|:------:|:----:|:-----:|:---------:|
+| Accurate time/durations results         |   ✓  |   ✓  |        ✓       |    ✓   |   ✓  |   ✓   |     ✗     |
+| Structured stack traces                 |   ✓  |   ✓  |        ✓       |    ✓   |   ✓  |   ✓   |     ✗     |
+| Distributed traces on Integration tests |   ✓  |   ✓  |        ✓       |    ✓   |   ✓  |   ✓   |     ✗     |
+| Agent based reporting                   |   ✓  |   ✓  |        ✓       |    ✓   |   ✓  |   ✓   |     ✗     |
+| Agentless reporting                     |   ✓  |   ✓  |        ✓       |    ✓   |   ✗  |   ✓   |     ✓     |
+| Test suite level visibility             |   ✓  |   ✓  |        ✓       |    ✓   |   ✗  |   ✓   |     ✓     |
+| Manual API                              |   ✓  |   ✓  |        ✓       |    ✗   |   ✗  |   ✓   |     ✗     |
+| Codeowner by test                       |   ✓  |   ✓  |        ✓       |        |   ✗  |   ✓   |     ✗     |
+| Source code start/end reporting         |   ✓  |   ✓  | ✓ (only start) |        |   ✗  |   ✓   |     ✗     |
+| CI and Git info                         |   ✓  |   ✓  |        ✓       |    ✓   |   ✓  |   ✓   |     ✓     |
+| Git metadata upload                     |   ✓  |   ✓  |        ✓       |    ✓   |   ✗  |   ✓   |     ✓     |
+| Intelligent Test Runner                 |   ✓  |   ✓  |        ✓       |    ✓   |   ✗  |   ✓   |     ✗     |
+| Code coverage support                   |   ✓  |   ✓  |        ✓       |        |   ✗  |   ✓   |     ✗     |
+| Benchmark tests support                 |   ✓  |   ✗  |        ✗       |    ✓   |   ✗  |   ✓   |     ✗     |
+| Parameterized tests                     |   ✓  |   ✓  |        ✓       |    ✓   |   ✗  |   ✓   |     ✗     |
```

Comment:
```suggestion
| Accurate time/durations results         |   ✓  |   ✓  |        ✓       |    ✓   |   ✓  |   ✓   |     ✗     |
| Structured stack traces                 |   ✓  |   ✓  |        ✓       |    ✓   |   ✓  |   ✓   |     ✗     |
| Distributed traces on Integration tests |   ✓  |   ✓  |        ✓       |       ✗     |   ✓  |   ✓   |     ✗     |
| Agent based reporting                   |   ✓  |   ✓  |        ✓       |    ✓   |   ✓  |   ✓   |     ✗     |
| Agentless reporting                     |   ✓  |   ✓  |        ✓       |    ✓   |   ✗  |   ✓   |     ✓     |
| Test suite level visibility             |   ✓  |   ✓  |        ✓       |    ✓   |   ✗  |   ✓   |     ✓     |
| Manual API                              |   ✓  |   ✓  |        ✓       |    ✗   |   ✗  |   ✓   |     ✗     |
| Codeowner by test                       |   ✓  |   ✓  |        ✓       |   ✗  |   ✗  |   ✓   |     ✗     |
| Source code start/end reporting         |   ✓  |   ✓  | ✓ (only start) |   ✗  |   ✗  |   ✓   |     ✗     |
| CI and Git info                         |   ✓  |   ✓  |        ✓       |    ✓   |   ✓  |   ✓   |     ✓     |
| Git metadata upload                     |   ✓  |   ✓  |        ✓       |    ✓   |   ✗  |   ✓   |     ✓     |
| Intelligent Test Runner                 |   ✓  |   ✓  |        ✓       |    ✓   |   ✗  |   ✓   |     ✗     |
| Code coverage support                   |   ✓  |   ✓  |        ✓       |    ✗  |   ✗  |   ✓   |     ✗     |
| Benchmark tests support                 |   ✓  |   ✗  |        ✗       |    ✓   |   ✗  |   ✓   |     ✗     |
| Parameterized tests                     |   ✓  |   ✓  |        ✓       |    ✓   |   ✗  |   ✓   |     ✗     |
```

Reply:
done in [8c48292](https://github.com/DataDog/documentation/pull/19793/commits/8c4829261e3bd92e0ac78fee7bf69ed046e5b49b)